include/dpp/message.h

```diff
@@ -839,6 +839,13 @@ struct DPP_EXPORT sticker : public managed {
 	 */
 	std::string build_json(bool with_id = true) const;
 
+	/**
+	 * @brief Get the sticker url
+	 *
+	 * @return std::string icon url
+	 */
+	std::string get_sticker_url() const;
```

Comment:
wouldn't `get_url()` be a better method name, as its obvious what url you're getting - its a method of the `sticker` object 😃

Reply:
i agree but i tried to follow the current style of all the other url getters

Reply:
1

Reply:
2